CRPL/donations.yml

```diff
@@ -113,7 +113,7 @@
   timestamp: 1652007659
 
   files:
-     - CRPL/Certificazione_Fiscale_20220508.png
+     - CRPL/Certificazione Fiscale_20220508_367425.png
```

Comment:
The PNG filename and the name here in the YAML file can't contain spaces. If you can update the filename in both places I can continue with the PR.

Reply:
Done. I'll take in to account on the next update. Thanks and have a nice day.